piqe_ocp_lib/tests/resources/test_ocp_nodes.py

```diff
@@ -224,3 +224,81 @@ def test_execute_command_on_a_node(self, setup_params):
             decoded_str = command_api_response[2].decode("utf-8")
             logger.info("Decoded stderr string {}".format(decoded_str))
             assert decoded_str.find("executable file not found in $PATH") >= 1
+
+    def test_get_all_master_nodes(self, setup_params):
+        """
+        Verify that a list of all master nodes is returned
+        1. Call get_master_nodes method
+        2. Verify that the response object is of kind NodeList
+        :param setup_params:
+        :return:
+        """
+        node_api_obj = setup_params["node_api_obj"]
+        # Get all master nodes
+        api_response = node_api_obj.get_master_nodes()
+        node_count = len(api_response.items)
+        logger.info("{} master nodes returned in the list".format(node_count))
+        assert api_response.kind == "NodeList"
+
+    def test_get_all_worker_nodes(self, setup_params):
+        """
+        Verify that a list of all worker nodes is returned
+        1. Call get_worker_nodes method
+        2. Verify that the response object is of kind NodeList
+        :param setup_params:
+        :return:
+        """
+        node_api_obj = setup_params["node_api_obj"]
+        # Get all worker nodes
+        api_response = node_api_obj.get_worker_nodes()
+        node_count = len(api_response.items)
+        logger.info("{} worker nodes returned in the list".format(node_count))
+        assert api_response.kind == "NodeList"
+
+    def test_is_node_schedulable(self, setup_params):
+        """
+        Verify that a boolean is returned based on the node schedulable status
+        1. Call get_all_node_names method via a ocp_nodes instance
+        2. Call is_node_schedulable method via a ocp_nodes instance
+        3. Verify that a True is returned if node unschedulable status is false
+        :param setup_params:
+        :return:
+        """
+        node_api_obj = setup_params["node_api_obj"]
+        # Get all master nodes
+        api_response_node_name_list = node_api_obj.get_all_node_names()
```

Comment:
Shouldn't 'get_master_nodes' be called here instead of 'get_all_node_name'? If I am reading this test correctly you are getting all node names then asserting that they are all schedulable. If that's the case, then it would seem that we assume all nodes are schedulable, which seems like it would make this test fail eventually.
I can see how this would always work if you sandwhiched 'test_is_node_schedulable' between 'test_mark_node_schedulable' and 'test_mark_node_unschedulable'. Both those tests seem to mark all nodes.